sites/cnfpt/CHANGELOG.md

```diff
@@ -8,8 +8,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [1.1.0] - 2021-03-05
+
 ### Added
+
 - Use custom views to handle errors (400, 403, 404, 500)
+- Enable LTIConsumerPlugin on `course_teaser` placeholder
```

Comment:
You missed this settings: https://github.com/openfun/richie/blob/master/sandbox/settings.py#L319

I propose configuring the LTI consumers directly in the code here and leave on the secrets to matsuo... (for the moment we have only one LTI service to configure: marsha).